src/testdrive.F90

```diff
@@ -154,6 +153,34 @@ module testdrive
   integer, parameter :: skipped = 77
 
 
+  !> ANSI codes for color support
+  character(len=*), parameter :: ESC = char(27)
+
+  !> Escape code for erasing current line
+  ! character(len=*), parameter :: LINE_RESET = ESC//"[2K"//ESC//"[1G"
+
+  !> Escape code for moving up one line
+  ! character(len=*), parameter :: LINE_UP = ESC//"[1A"
+
+  !> Escape code for moving down one line
+  ! character(len=*), parameter :: LINE_DOWN = ESC//"[1B"
```

Comment:
Do we need to keep the commented out code? Likewise for COLOR_YELLOW below.

Reply:
I agree, they should be removed 👍